partcad-cli/src/partcad_cli/click/commands/list/sketches.py

```
@@ -21,20 +21,27 @@
         return
     package = package_obj.name
 
-    with Process("ListSketches", package):
+    with pc_logging.Process("ListSketches", package):
         sketch_count = 0
         sketch_kinds = 0
 
         if used_by is not None:
-            logging.info("Instantiating %s..." % used_by)
-            ctx.get_assembly(used_by)
+            pc_logging.info("Instantiating %s..." % used_by)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)